.estafette.yaml

```diff
@@ -150,6 +150,33 @@ releases:
           memory:
             request: 34Mi
             limit: 68Mi
+        - type: git-sync
+          image: k8s.gcr.io/git-sync:v3.1.5
+          env:
+          - name: GIT_KNOWN_HOSTS
+            value: "false"
+          - name: GIT_SYNC_MAX_SYNC_FAILURES
+            value: "10"
+          - name: GIT_SYNC_WAIT
+            value: "60"
+          - name: GIT_SYNC_ROOT
+            value: "/git-sync-root"
+          args:
+          - "-ssh"
+          - estafette.secret(qy7o7SKbzOL1NSTf.nFNX_QO2JiVO-WzIq2ory58Fu7rcFA5Q6PiAMZ5j2WA2csTFe-DDdFVe-_EauA4ocGdHGZO0R8-08rCbb7UdOq8c5lQ1jxIK.2RdR8Aa7NSFI4APPsWoo2JkaqqvdHg9Ds-aMM4t0gCw6LNHUdJTEW4oO47nVON4JHmFYhlY=)
+          - "-dest=/configs"
+          - "-branch=master"
+          - "-depth=1"
+          - "-webhook-url=http://127.0.0.1:9090/-/reload"
```

Comment:
This isn't needed; estafette-ci-api detects if config files change automatically.